index.html

```diff
@@ -2250,6 +2329,36 @@ <h1>Core Representations</h1>
 for more information.
     </p>
 
+    <section>
+      <h2>Creating Representations</h2>
+      <p>
+The <a href="#data-model">data model</a> provided in this specification
+supports being serialized into a variety of existing representations. Some
+applications might require the creation of a new representation. All
+representations require the following:
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding for all
+property names and their associated values as defined in this specification.
+This enables anything that can be represented in the <a>DID document</a> data
+model to also be represented in a compliant representation.
+        </li>
+        <li>
+The representation MUST be associated with an IANA-registered MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with the fragment processing rules defined in section
+<a href="#fragment"></a> of this specification.
+        </li>
+      </ol>
```

Comment:
The representation must also define default generic type processing rules for consumption of all data types within the representation.

Reply:
I'm not understanding how this differs from a representation defining unambiguous encoding and decoding for all property names and the data types of their associated values, could you clarify?

Reply:
I think I understand what @jricher is getting at, and that's planned for a future PR. Basically, each representation must list all the abstract data model data types and how you express those concretely in the representation. The representation sections kinda/sorta have this right now, but if this PR goes in, we'll be able to be far more explicit and prescriptive about those serializations in each representation.

Reply:
@msporny has the right of it, but it also needs to clearly go both ways, to and from the data model data types and the representation's data types. So you need things like this strawman:

"If you see an Integer in the ADM it goes in a JSON Number"

As well as:

"If you see a JSON Number in the representation, and don't have an additional property definition, it is an ADM Float".

I'm ok with this being in a separate PR but this can't be lost.